config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: quay.io/kuadrant/multicluster-gateway-controller
+  newTag: main
```

Comment:
Not strictly necessary for the quick start script to work since you have another kustomization that sets it elsewhere. There's no issue changing this, probably makes more sense, but one thing that might happen now is this file being changed after running `make deploy-controller`. You might want to look at making that use the same image and tag names.

Reply:
yes that make command uses `controller:latest` I chose not to change that one because it would keep the make command consistent with it's previous behaviour. let me know though if i should go ahead and change that? I think it would be just removing the edit like you had in the example?

Reply:
We can come back to it if it becomes an issue.